notifier/__main__.py

```diff
@@ -1,5 +1,37 @@
-from .core.notifierapp import NotifierApp
+import argparse
+from .core.celeryapp import CeleryApp
+from .core.flaskapp import FlaskApp
+
+
+def args():
+    args = argparse.ArgumentParser(epilog=__doc__)
```

Comment:
All this implementation must be in a different directory probably the best location is ```notifier/cli/```